pkg/util/runtime/features.go

```diff
@@ -53,21 +53,25 @@ const (
 
 	// FeatureCustomFasSyncInterval is a feature flag that enables custom the FleetAutoscaler rsync interval
 	FeatureCustomFasSyncInterval Feature = "CustomFasSyncInterval"
+
+	// FeatureGracefulTerminationFilter is a feature flag that enables SDK to support gracefulTermination
+	FeatureGracefulTerminationFilter Feature = "GracefulTerminationFilter"
```

Comment:
Not sure what the `Filter` part is alluding too here.

Maybe `FeatureSDKGracefulTermination` ?